lib/configuration.js

```diff
@@ -229,6 +223,21 @@ Configuration.prototype._checkLocalServiceContext = function() {
  * @returns {Undefined} - does not return anything
  */
 Configuration.prototype._gatherLocalConfiguration = function() {
+  if (this._givenConfiguration.ignoreEnvironmentCheck === true) {
+    this._shouldReportErrorsToAPI = true;
+  } else if (has(this._givenConfiguration, 'ignoreEnvironmentCheck')) {
+    throw new Error('config.ignoreEnvironmentCheck must be a boolean');
+  } else {
+    this._shouldReportErrorsToAPI = env.NODE_ENV === 'production';
+  }
+  if (!this._shouldReportErrorsToAPI) {
+    this._logger.warn([
+      'Stackdriver error reporting client has not been configured to send',
+      'errors, please check the NODE_ENV environment variable and make sure it',
+      'is set to "production" or the ignoreEnvironmentCheck property is set to',
+      '"true" in the runtime configuration object'
```

Comment:
nit: drop the double quotes around true as people might set it to the string "true".